### PR TITLE
Initial commit of amp-inputmask example

### PR DESCRIPTION
**Pull request by cvializ**

### CODE REVIEW

File: src/20_Components/amp-inputmask.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!---{
+    "experiments": ["amp-inputmask"],
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  Add input masks to inputs in your amp-forms. At this time, amp-inputmask
```
Comment (kul3r4) [inline src/20_Components/amp-inputmask.html]: add "`" to amp-inputmask and possibly everywhere yoy use it in the docs below

File: src/20_Components/amp-inputmask.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <meta charset="utf-8">
+  <title>amp-inputmask</title>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-inputmask/">
+  <!-- ## Setup -->
+  <!-- Import the amp-inputmask component in the header -->
+  <script async custom-element="amp-inputmask" src="https://cdn.ampproject.org/v0/amp-inputmask-0.1.js"></script>
+  <!-- Also import the amp-form component in the header -->
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
```
Comment (kul3r4) [inline src/20_Components/amp-inputmask.html]: can you add why you are importing amp-mustache

File: src/20_Components/amp-inputmask.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<!---{
+    "experiments": ["amp-inputmask"],
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  Add input masks to inputs in your amp-forms. At this time, amp-inputmask
+  also requires the amp-form extension.
```
Comment (sebastianbenz) [inline src/20_Components/amp-inputmask.html]: Can you add why this is useful?

File: src/20_Components/amp-inputmask.html [hunk 4]
```diff
@@ -34,7 +46,7 @@
 
      In the mask, `L` allows the user to enter an alphabetic character.
      `0` denotes a numeric character. `_` is a literal space character.
-     See the full mask specification at [the amp-inputmask documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-inputmask/README.md)
+     See the full mask specification at [the `amp-inputmask` documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-inputmask/README.md)
```
[the remark this answers is not in the document]
Comment (sebastianbenz) [reply src/20_Components/amp-inputmask.html]: Looks like it hasn't been deployed yet. Github link is fine in that case, but we need to remember to replace it once it's live on ampproject.org.

[the remark this answers is not in the document]
Comment (cvializ) [reply src/20_Components/amp-inputmask.html]: Cool, I'll make an issue to update it once the docs updates are deployed

[the remark this answers is not in the document]
Comment (cvializ) [reply src/20_Components/amp-inputmask.html]: https://github.com/ampproject/amp-by-example/issues/1612

File: src/20_Components/amp-inputmask.html [hunk 5]
```diff
@@ -5,8 +5,13 @@
...
 
-  Add input masks to inputs in your amp-forms. At this time, amp-inputmask
-  also requires the amp-form extension.
+  Input masks automatically add formatting characters to user input, and prevent
+  users from typing input that doesn't match the mask.
+  For example, an input mask on a telephone field automatically adds
+  special characters like `(`, `)` and `-`, and users can type only the numbers
+  needed while the mask prevents them from typing incorrect characters.
+
+  `amp-inputmask` enables input masks for form fields in AMP documents.
```
[the remark this answers is not in the document]
Comment (cvializ) [reply src/20_Components/amp-inputmask.html]: Added temporary link to github md

**Comment by cvializ** [comment]
PTAL

File: backend/amp-inputmask.go [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import (
+	"net/http"
+)
+
+const (
+	AMP_INPUTMASK_SAMPLE_NAME = "/" + CATEGORY_COMPONENTS + "/amp-inputmask/"
+)
+
+func InitAmpInputmask() {
+	RegisterHandler(AMP_INPUTMASK_SAMPLE_NAME+"postal", onlyPost(submitFormXHRInputMask))
```
[the remark this answers is not in the document]
Comment (cvializ) [reply backend/amp-inputmask.go]: Oops, refactored and missed one of the `action-xhr`s

**Comment by cvializ** [comment]
Thank you very much for the review!

**Comment by usernamefghh** [comment]
on

File: src/20_Components/amp-inputmask.html (hunk 1 above)
Comment (usernamefghh) [reply src/20_Components/amp-inputmask.html]: ago

File: src/20_Components/amp-inputmask.html (hunk 3 above)
Comment (usernamefghh) [reply src/20_Components/amp-inputmask.html]: vializ

**Pull request merged**